From 2d74861d9388ba250148b652f9a7edd171902c15 Mon Sep 17 00:00:00 2001
From: Gabriel Craciunescu <crazy@frugalware.org>
Date: Sat, 26 Aug 2017 06:49:54 +0200
Subject: [PATCH] packages: make it work for now  only '
 pkgman.install(entry[key]) ' works .. not sure what broke the rest  so use
 this workaround for now

---
 src/modules/packages/main.py | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/src/modules/packages/main.py b/src/modules/packages/main.py
index 93fcc9eb7..8a035afe5 100644
--- a/src/modules/packages/main.py
+++ b/src/modules/packages/main.py
@@ -154,13 +154,8 @@ def run_operations(pkgman, entry):
     for key in entry.keys():
         entry[key] = subst_locale(entry[key])
         if key == "install":
-            if isinstance(entry[key], list):
-                for package in entry[key]:
-                    pkgman.run(package["pre-script"])
-                    pkgman.install([package["package"]])
-                    pkgman.run(package["post-script"])
-            else:
-                pkgman.install(entry[key])
+            # not sure what changed but only that work for now
+            pkgman.install(entry[key])
         elif key == "try_install":
             # we make a separate package manager call for each package so a
             # single failing package won't stop all of them
-- 
2.14.1

